src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -190,6 +193,7 @@ describe('Exam assessment', () => {
         });
 
         beforeEach('Create exercise and submission', () => {
+            cy.login(admin);
```

Comment:
why is an admin login necessary here?
Creating exercises should done by instructors